apps/api/src/manifest/manifest.service.spec.ts

```diff
@@ -0,0 +1,242 @@
+import { ManifestService } from './manifest.service';
+import { HttpService } from '@nestjs/axios';
+import { AxiosResponse } from 'axios';
+import { of } from 'rxjs';
+import { TildaManifest } from '../models';
+import { Test, TestingModule } from '@nestjs/testing';
+
+describe('ManifestService', () => {
+  const mockManifest = {
```

Comment:
We have it as TildaManifest, can we turn it into a fixture and use it in tests? @mehmetsagdur